mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -124,6 +124,71 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomJitterPoints(object):
+    """Randomly jitter point coordinates.
+
```

Comment:
Should add a description for this module in detail, e.g., the noise applied to different points are different.

Reply:
Yes, I have added a description.